src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCaller.java

```diff
@@ -0,0 +1,232 @@
+ *
+ * <p>Finally, HaplotypeCaller is also able to correctly handle the splice junctions that make RNAseq a challenge for most variant callers.</p>
+ *
+ * <h3>How HaplotypeCaller works</h3>
+ *
+ * <br />
+ * <h4>1. Define active regions </h4>
+ *
+ * <p>The program determines which regions of the genome it needs to operate on, based on the presence of significant
+ * evidence for variation.</p>
+ *
+ * <br />
+ * <h4>2. Determine haplotypes by assembly of the active region </h4>
+ *
+ * <p>For each ActiveRegion, the program builds a De Bruijn-like graph to reassemble the ActiveRegion, and identifies
+ * what are the possible haplotypes present in the data. The program then realigns each haplotype against the reference
+ * haplotype using the Smith-Waterman algorithm in order to identify potentially variant sites. </p>
+ *
+ * <br />
+ * <h4>3. Determine likelihoods of the haplotypes given the read data </h4>
+ *
+ * <p>For each ActiveRegion, the program performs a pairwise alignment of each read against each haplotype using the
+ * PairHMM algorithm. This produces a matrix of likelihoods of haplotypes given the read data. These likelihoods are
+ * then marginalized to obtain the likelihoods of alleles for each potentially variant site given the read data.   </p>
+ *
+ * <br />
+ * <h4>4. Assign sample genotypes </h4>
+ *
+ * <p>For each potentially variant site, the program applies Bayes' rule, using the likelihoods of alleles given the
+ * read data to calculate the likelihoods of each genotype per sample given the read data observed for that
+ * sample. The most likely genotype is then assigned to the sample.    </p>
+ *
+ * <h3>Input</h3>
+ * <p>
+ * Input bam file(s) from which to make calls
+ * </p>
+ *
+ * <h3>Output</h3>
+ * <p>
+ * Either a VCF or gVCF file with raw, unfiltered SNP and indel calls. Regular VCFs must be filtered either by variant
+ * recalibration (best) or hard-filtering before use in downstream analyses. If using the reference-confidence model
+ * workflow for cohort analysis, the output is a GVCF file that must first be run through GenotypeGVCFs and then
+ * filtering before further analysis.
+ * </p>
+ *
+ * <h3>Usage examples</h3>
+ *
+ * <p>These are example commands that show how to run HaplotypeCaller for typical use cases. Square brackets ("[ ]")
+ * indicate optional arguments. Note that parameter values shown here may not be the latest recommended; see the
+ * Best Practices documentation for detailed recommendations. </p>
+ *
+ * <br />
+ * <h4>Single-sample GVCF calling on DNAseq (for `-ERC GVCF` cohort analysis workflow)</h4>
+ * <pre>
+ *   java -jar GenomeAnalysisTK.jar \
```

Comment:
How about we wait until gatk-launch is ported to protected before updating these? We don't want to have to update the docs more than once...